libs/design-system/src/lib/Components/Footer/index.tsx

```diff
@@ -0,0 +1,170 @@
+import { ReactComponent as ForumIcon } from '../../Images/social/forum-light.svg';
+import { ReactComponent as OnFlowIcon } from '../../Images/logos/flow-icon-bw-light.svg';
+
+// reduce repetition of the section layout in Footer component
+const footerSections = [
+  {
+    header: 'Documentation',
+    links: [
+      {
+        link: '#', // TODO: replace with actual links
+        text: 'Getting Started',
+      },
+      {
+        link: '#',
+        text: "SDK's",
+      },
+      {
+        link: '#',
+        text: 'Tutorials',
+      },
+      {
+        link: '#',
+        text: 'Guides',
+      },
+      {
+        link: '#',
+        text: 'Concepts',
+      },
+      {
+        link: '#',
+        text: 'Walkthroughs',
+      },
+      {
+        link: '#',
+        text: 'How-To',
+      },
+      {
+        link: '#',
+        text: 'FAQ',
+      },
+      {
+        link: '#',
+        text: 'Tools',
+      },
+    ],
+  },
+  {
+    header: 'Community',
+    links: [
+      {
+        link: '#',
+        text: 'Ecosystem',
+      },
+      {
+        link: '#',
+        text: 'GitHub',
+      },
+      {
+        link: '#',
+        text: 'Discord',
+      },
+      {
+        link: '#',
+        text: 'Flowverse',
+      },
+      {
+        link: '#',
+        text: 'Forum',
+      },
+    ],
+  },
+  {
+    header: 'Start Building',
+    links: [
+      {
+        link: '#',
+        text: 'Flow Playground',
+      },
+      {
+        link: '#',
+        text: 'Crypto Kitties',
+      },
+    ],
+  },
+  {
+    header: 'Network',
+    links: [
+      {
+        link: '#',
+        text: 'Network Status',
+      },
+      {
+        link: '#',
+        text: 'Mainnet',
+      },
+      {
+        link: '#',
+        text: 'Testnet',
+      },
+      {
+        link: '#',
+        text: 'Canary',
+      },
+      {
+        link: '#',
+        text: 'Sporks FAQ',
+      },
+    ],
+  },
+];
+
+export function Footer() {
+  return (
+    <footer className="bg-black px-6 text-white">
+      <div className="container mx-auto">
+        <div className="block px-2 pt-8 pb-6 md:flex md:items-center md:justify-between md:px-4 md:pt-16">
+          <div>
+            <FlowDocsLogo className="h-10 w-min object-contain" />
+          </div>
+          <div className="flex items-center justify-start pt-8 lg:justify-center lg:pt-0">
+            <a href="#" className="px-3">
+              <GithubIcon fill="red" height={32} width={32} />
+            </a>
+            <a href="#" className="px-3">
+              <DiscordIcon fill="white" height={28} width={28} />
+            </a>
+            <a href="#" className="px-3">
+              <ForumIcon fill="white" height={24} width={24} />
+            </a>
+            <a href="#" className="px-3">
+              <OnFlowIcon height={28} width={28} />
+            </a>
+            <a href="#" className="pl-3">
+              <Link to="#">Eng</Link>
```

Comment:
No need for the <Link> here

Reply:
I think I might need some clarification for this comment, not quite sure what its referring to :)

Reply:
@neilZon hah the github markdown parsed out the `<Link>` from my comment - updated :)